Index: app/src/main/java/com/example/wikidota/MobileActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/wikidota/MobileActivity.java b/app/src/main/java/com/example/wikidota/MobileActivity.java
--- a/app/src/main/java/com/example/wikidota/MobileActivity.java	
+++ b/app/src/main/java/com/example/wikidota/MobileActivity.java	
@@ -4,10 +4,8 @@
 
 import android.content.Intent;
 import android.os.Bundle;
-import android.util.Log;
 import android.view.View;
 
-import com.example.wikidota.HeroActivity;
 public class MobileActivity extends AppCompatActivity {
 
     @Override
@@ -89,9 +87,8 @@
         Intent intent = new Intent(MobileActivity.this, HeroActivity.class);
         intent.putExtra("name", "enigma");
         startActivity(intent);
-
+    }
 
-    }
     public void getio(View view) {
         Intent intent = new Intent(MobileActivity.this, HeroActivity.class);
         intent.putExtra("name", "io");
Index: app/src/main/java/com/example/wikidota/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/wikidota/MainActivity.java b/app/src/main/java/com/example/wikidota/MainActivity.java
--- a/app/src/main/java/com/example/wikidota/MainActivity.java	
+++ b/app/src/main/java/com/example/wikidota/MainActivity.java	
@@ -2,7 +2,6 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
-import android.annotation.SuppressLint;
 import android.content.Intent;
 import android.os.AsyncTask;
 import android.os.Bundle;
